### PR TITLE
Added my project

### PR DESCRIPTION
**Pull request by hemantwasthere**

### CODE REVIEW

File: public/Projects/Hemant/Portfolio/code/public/index.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  <script type="text/javascript">
+    (function () {
+      // https://dashboard.emailjs.com/admin/account
+      emailjs.init('XNV7d16a8nY806xbo');
+    })();
+
+  </script>
+</body>
+
+</html>
```
Comment (Panquesito7) [inline public/Projects/Hemant/Portfolio/code/public/index.html:41]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
There are a few other files missing an endline as well; you might want to fix that up. 🙂

```suggestion
</html>

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/Projects/Hemant/Portfolio/code/public/index.html (hunk 1 above)
Comment (hemantwasthere) [reply public/Projects/Hemant/Portfolio/code/public/index.html:41]: Yeah sure, i'll be doing that

File: public/Projects/Hemant/Portfolio/code/.prettierrc [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": false,
+    "singleQuote": true
+  }
```
Comment (Panquesito7) [inline public/Projects/Hemant/Portfolio/code/.prettierrc:6]: This file is missing an endline. A few other files are missing an endline as well.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: public/Projects/Hemant/Portfolio/code/.prettierrc (hunk 2 above)
Comment (hemantwasthere) [reply public/Projects/Hemant/Portfolio/code/.prettierrc:6]: Ohh okay so i have to do this in every file.
Got it

File: public/Projects/Hemant/Portfolio/code/src/App.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  left: 0;
+  right: 0;
+  margin: auto;
+  top: 0;
+  bottom: 0;
+  width: 50px;
+  height: 50px;
+  animation: fadeOut 1s 1s;
+  animation-fill-mode: forwards;
+}
```
Comment (Panquesito7) [inline public/Projects/Hemant/Portfolio/code/src/App.scss:54]: Missing an endline here.
```suggestion
}

```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

File: public/Projects/Hemant/Portfolio/code/.prettierrc [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (Panquesito7) [inline public/Projects/Hemant/Portfolio/code/.prettierrc:7]:
```suggestion
}
```


__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inelegant

**Comment by hemantwasthere** [comment]
I'll open another PR, It got some issues

**Pull request closed without merging**